lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+    "description": "Header of file actions for file"
+  },
+  "file-already-exists": {
+    "defaultMessage": "Aquest fitxer ja existeix",
+    "description": "File already exists notification text"
+  },
+  "file-name-invalid-character": {
+    "defaultMessage": "El nom conté un caràcter no vàlid: {invalid}",
+    "description": "Warning text for new Python file name with an invalid character"
+  },
+  "file-name-length": {
+    "defaultMessage": "El nom és massa llarg",
+    "description": "Warning text for overly long file names (file system limitation)"
+  },
+  "file-name-lowercase-only": {
+    "defaultMessage": "El nom ha de ser tot en minúscula",
+    "description": "Warning text for new Python file with uppercase"
+  },
+  "file-name-not-empty": {
+    "defaultMessage": "El nom no pot estar buit",
+    "description": "Warning text for new Python file with empty name"
+  },
+  "file-name-start-number": {
+    "defaultMessage": "El nom no pot començar amb un nombre",
+    "description": "Warning text for new Python file names starting with a number"
+  },
+  "file-name-whitespace": {
+    "defaultMessage": "El nom no pot contenir espais",
+    "description": "Warning text for new Python file name with whitespace"
+  },
+  "firmware-update-link": {
+    "defaultMessage": "Has d'actualitzar el teu microprogramari <link>update your firmware</link> abans de poder connectar-te a aquesta micro:bit.",
+    "description": "Text in the firmware update dialog"
+  },
+  "firmware-update-message": {
+    "defaultMessage": "La connexió a la micro:bit ha fallat perquè el microprogramari de la teva micro:bit és massa antic.",
+    "description": "Text in the firmware update dialog"
+  },
+  "firmware-update-title": {
+    "defaultMessage": "Cal actualitzar el firmware",
+    "description": "Title for the firmware update dialog"
+  },
+  "flash-action": {
+    "defaultMessage": "Transferir",
+    "description": "Text for flash button"
+  },
+  "flash-hover": {
+    "defaultMessage": "Transfereix el projecte directament a la micro:bit",
+    "description": "Hover text over flash button"
+  },
+  "flashing-code": {
+    "defaultMessage": "S'està enviant el programa (transferint)",
+    "description": "Progress dialog text for partial flashes (quick)"
+  },
+  "flashing-full-flash-detail": {
+    "defaultMessage": "El teu programa s'està enviant (transferint) a la teva micro:bit. Això pot trigar una mica la primera vegada, però després serà més ràpid.",
+    "description": "Message shown after the title in the progress dialog for a full flash"
+  },
+  "flashing-micropython": {
+    "defaultMessage": "Transferint MicroPython ",
+    "description": "Progress dialog text for full flashes (slower)"
+  },
+  "font-size": {
+    "defaultMessage": "Mida de la font",
+    "description": "Font size option text"
+  },
+  "guide-link": {
+    "defaultMessage": "Per obtenir més suport, consulta la guia de l'editor de Python <link>Python Editor guide </link>",
```

Comment:
Translate text in `<link>`, don't duplicate.

Reply:
Requested in Crowdin